src/app/features/event/event-comment/event-comment-single/event-comments-single.component.ts

```diff
@@ -0,0 +1,14 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { EventComment } from '../../event.interfaces';
+
+@Component({
+  selector: 'app-event-comments-single',
+  standalone: true,
+  templateUrl: './event-comments-single.component.html',
+  styleUrls: ['./event-comments-single.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class EventCommentsSingleComponent {
+  @Input()
+  comment!: EventComment;
```

Comment:
Jak dodajesz, że input jest required to można też to zadeklarować w selectorze, wtedy wywali nam błąd jak nie wprowadzimy tego inputa do komponentu w templatce: `app-event-comments-single[comment]`